### ISSUE TITLE
Compiling Less stalls after last update

### ISSUE THREAD

**Issue by zachleigh** [mrcrowl/vscode-easy-less#55]
Since the last update, compiling Less freezes and never completes. There is no error in the console and no output.

**Comment by mrcrowl** [comment]
Could you share a less file where this happens? I can’t repro from your description, unfortunately.

**Comment by mrcrowl** [comment]
In the meantime, you can install a previous version using the vsix files in the releases page of github.

**Comment by zachleigh** [comment]
Thanks for the quick reply @mrcrowl .

I dug around and found that we actually had a LESS error in one of our imported LESS files:
```
border: right: 1px solid #d1d1d1; // should be border-right: 1px solid #d1d1d1;
```

This caused the extension (or maybe LESS??) to freeze up. It would be nice if the extension exited with an error when this sort of thing happens. If that is beyond your control, understandable :)

**Comment by billreinhardt** [comment]
This hangs for me on "Compiling less --> css" as well, and I have to restart VScode to get it to stop. I am trying to go through all of my import files (there are tons) and see if there are any malformed rulesets, but is there any way we can get it to overlook issues as apparently happened in older versions?

**Comment by mrcrowl** [comment]
@billreinhardt, what OS and VS Code version?

**Comment by mrcrowl** [comment]
Also, do you have an isolated sample .less file where this happens?  Just struggling to repro this at my end.

**Comment by billreinhardt** [comment]
> @billreinhardt, what OS and VS Code version?

I tried it on both Windows (latest version 10) and on MacOS Mojave 10.14.6 using the latest versions of VS Code, 1.39

> Also, do you have an isolated sample .less file where this happens? Just struggling to repro this at my end.

Yes - see this file for details on what is going on:
https://www.ergotron.com/portals/0/media/downloads/details.txt

**Comment by billreinhardt** [comment]
@mrcrowl is this helpful for diagnosing the problem?

> > @billreinhardt, what OS and VS Code version?
> 
> I tried it on both Windows (latest version 10) and on MacOS Mojave 10.14.6 using the latest versions of VS Code, 1.39
> 
> > Also, do you have an isolated sample .less file where this happens? Just struggling to repro this at my end.
> 
> Yes - see this file for details on what is going on:
> https://www.ergotron.com/portals/0/media/downloads/details.txt

**Comment by mrcrowl** [comment]
Thanks for the repro example (and apologies for the long delay). Resolved in [v1.6.1](https://github.com/mrcrowl/vscode-easy-less/releases/tag/v1.6.1).
